### DISCUSSION TITLE
Windows form with App server

### DISCUSSION THREAD

**Discussion by babaauto** [MarimerLLC/csla#2043] [Questions]
Sorry,I am relatively new to Csla.net I have a windows app using NET 5 and an app server but when I run my windows app I am getting the following exceptions

1)HttpRequestException: The SSL connection could not be established, see inner exception.
2)AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot
I have followed the exact guide as per ProjectTracker Windows App.Is there something I am missing.

Your help will be much appreciated

**Comment by rockfordlhotka** [comment]
The easiest solution would be to change your aspnetcore app server to not require or use SSL on your dev machine.

I suspect that the other option is to run the aspnetcore app server project in the debugger so Visual Studio asks you to accept and add the test SSL cert as a trusted cert.
